pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd_migration_use-cases/guide.de-de.md

```diff
@@ -20,16 +20,21 @@ details[open]>summary::before {
 
 > [!primary]
 >
-> VCD on OVHcloud is currently in alpha phase. This guide may therefore be incomplete and can be updated in the future. 
-> 
+> VCD on OVHcloud is currently in alpha phase. This guide may therefore be incomplete and can be updated in the future.
+>
 
 ## Objective
 
 **The aim of this guide is to audit the special cases of vSphere managed HPC that may pose problems for migration to VCD on OVHcloud.**
 
 ## Requirements
 
+- Have a Hosted Private Cloud managed VMware product on OVHcloud.
 - You must have completed the verification steps in the following guide before launching a migration to VCD.
+- Do not use Zerto backup on OVHcloud solutions.
+- Do not require PCI-DSS, SNC or HDS certification.
+- Access to your managed vSphere [web interface](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vsphere_interface_connection).
+- Not having several datacentres with your managed vSphere solution on OVHcloud.
```

Comment:
Maybe remove this from the requirements section, because as a reader, if I have multiple DC, I will not read any further, and in the table you explain how to comply.